crates/xilem_svg/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
```

Comment:
Yep landing the svg branch first sounds sensible. IMO xilem is still early/experimental enough that landing stuff that isn't perfect (with a plan for how it will be improved) is fine.